install/kubecost/install.yml

```diff
@@ -0,0 +1,13 @@
+id: setup-kubecost-agent
```

Comment:
We are no longer accepting install plans. Please delete this file.

Reply:
To jump on Lucia's comment, we have replaced install-plans with data sources. So since you already have one of those, let's just remove this install plan! (we can just delete this file and remove the reference in the quickstart config)